ui/slider.js

```diff
@@ -564,6 +564,18 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		return this.max;
 	},
 
+	_refreshRange: function ( orientation ) {
+		if ( this.options.range === false ) {
```

Comment:
I thought to do so. I may be wrong but by this approach it will be necessary to write the such check if this function is invoked in another method. Correct me if  I'm missing something.